src/spyglass/micro_ct/v1/micro_ct.py

```diff
@@ -0,0 +1,120 @@
+import datajoint as dj
+
+from spyglass.common import (  # noqa: F401
+    AnalysisNwbfile,
+    CoordinateSystem,
+    LabMember,
+    Subject,
+)
+from spyglass.utils import SpyglassMixin, logger
+
+schema = dj.schema("microct_v1")
+
+
+@schema
+class MicroCTScan(SpyglassMixin, dj.Manual):
+    definition = """
+    # Metadata for a microCT scan of a subject's brain/tissue
+    -> Subject
+    scan_id: varchar(32)                # User-defined ID (e.g., 'SubjX_OsO4_Scan1')
+    ---
+    # --- Preparation Details ---
+    stain_reagent='Aqueous 2% OsO4': varchar(128) # Staining reagent details (e.g., 'OsO4', 'I2E')
+    stain_duration_days=14.0: float     # Duration of staining in days
+    embedding_resin='Durcupan ACM': varchar(128) # Resin used for embedding
+    prep_protocol_notes="": varchar(2048) # Notes on staining, dehydration, embedding variations
+
+    # --- Scan Details ---
+    scan_date=NULL: date                # Date of the scan itself
+    scanner_details: varchar(255)       # e.g., 'Nikon Metrology X-Tek HMX ST 225 @ HCNS'
+    source_target_type='Molybdenum': varchar(64) # X-ray source target material
+    filter_material='None': varchar(64) # Filter material used (e.g., 'None', 'Copper')
+    filter_thickness_mm=0.0: float      # (mm) Filter thickness
+    voltage_kv: float                   # (kV) Scan voltage (e.g., 100, 130)
+    current_ua: float                   # (uA) Scan current (e.g., 105, 135)
+    exposure_time_s: float              # (s) Exposure time per projection (e.g., 1.0)
+    num_projections: int                # Number of projections acquired (e.g., 3184)
+    frames_per_projection=1: int        # Frames averaged per projection (e.g., 1, 4)
+
+    # --- Reconstruction Details ---
+    reconstruction_software=NULL: varchar(128) # e.g., 'CT Pro (Nikon)', 'NRecon (Bruker)'
+    voxel_size_x: float                 # (um) Reconstructed voxel size X
+    voxel_size_y: float                 # (um) Reconstructed voxel size Y
+    voxel_size_z: float                 # (um) Reconstructed voxel size Z
+    output_format='TIFF stack': varchar(64) # Format of raw reconstructed data
+
+    # --- Data Location & Notes ---
+    raw_scan_path: varchar(512)         # Path to raw output (e.g., folder containing TIFF stack)
```

Comment:
This PR includes a few cases of varchar instead of filepath. We seem to only use the `filepath` attr for nwb files. It adds some complexity to the config, but saves us a lot of cleanup time with various cleanup functions that look for orphaned files. Moving forward, I think I'd advocate for varchars only in cases where we expect the file to change and don't want to update the checksum